js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,33 @@
+
+function leastCommonMultiple(...args) {
+  let store = [], result = [];
+  let sortedArray =  args.sort(function(a,b) { 
+    return a - b; 
+  });
+
+  let i =2;
```

Comment:
use some visual code extension like Prettier for proper spacing. Here` let i =2;` there should be a space between operator and number like this ` let i = 2;`